package.json

```diff
@@ -1,68 +1,52 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "lint:fix": "eslint src --fix && pnpm format",
-    "lint:strict": "eslint --max-warnings=0 src",
+    "lint:fix": "next lint --fix",
     "typecheck": "tsc --noEmit --incremental false",
     "format": "prettier -w .",
-    "format:check": "prettier -c .",
-    "prepare": "husky install"
+    "format:check": "prettier -c ."
   },
   "dependencies": {
     "@headlessui/react": "1.7.13",
     "@headlessui/tailwindcss": "0.1.2",
-    "@lukemorales/query-key-factory": "1.2.0",
-    "@rainbow-me/rainbowkit": "0.12.4",
-    "@tanstack/react-query": "4.28.0",
-    "clsx": "1.2.1",
-    "dayjs": "1.11.7",
-    "ethers": "5",
-    "next": "13.2.3",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "@lukemorales/query-key-factory": "1.3.4",
+    "@rainbow-me/rainbowkit": "2.2.0",
+    "@tanstack/react-query": "5.59.20",
+    "clsx": "2.1.1",
+    "dayjs": "1.11.13",
+    "next": "14.2.17",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
     "react-icons": "4.8.0",
-    "tailwind-merge": "1.10.0",
-    "wagmi": "0.12.6",
-    "zustand": "4.3.6"
+    "tailwind-merge": "2.5.4",
+    "viem": "2.21.42",
+    "wagmi": "2.12.27",
+    "zustand": "5.0.1"
   },
   "devDependencies": {
     "@graphprotocol/client-cli": "2.2.20",
-    "@svgr/webpack": "6.5.1",
-    "@types/node": "18.15.0",
-    "@types/react": "18.0.28",
-    "@typescript-eslint/eslint-plugin": "5.56.0",
-    "@typescript-eslint/parser": "5.56.0",
-    "autoprefixer": "10.4.13",
-    "cssnano": "5.1.15",
-    "eslint": "8.35.0",
-    "eslint-config-next": "13.2.3",
-    "eslint-config-prettier": "8.8.0",
-    "eslint-plugin-simple-import-sort": "7.0.0",
-    "eslint-plugin-unused-imports": "2.0.0",
-    "husky": "7.0.4",
-    "lint-staged": "12.5.0",
-    "postcss": "8.4.21",
-    "prettier": "2.8.6",
-    "prettier-plugin-tailwindcss": "0.1.13",
-    "tailwindcss": "3.2.7",
-    "typescript": "4.9.5",
-    "zod": "3.21.4"
+    "@svgr/webpack": "8.1.0",
+    "@types/node": "22.9.0",
+    "@types/react": "18.3.12",
+    "autoprefixer": "10.4.20",
+    "cssnano": "7.0.6",
+    "eslint": "^8",
+    "eslint-config-next": "14.2.17",
+    "postcss": "8.4.47",
+    "prettier": "3.3.3",
+    "prettier-plugin-tailwindcss": "0.6.8",
+    "tailwindcss": "3.4.14",
+    "typescript": "5.6.3"
   },
-  "lint-staged": {
-    "**/*.{js,jsx,ts,tsx} !.prettierrc.js": [
-      "eslint --max-warnings=0",
-      "prettier -w"
-    ],
-    "**/*.{json,css,scss,md,webmanifest}": [
-      "prettier -w"
-    ]
-  },
-  "packageManager": "pnpm@9.10.0+sha512.73a29afa36a0d092ece5271de5177ecbf8318d454ecd701343131b8ebc0c1a91c487da46ab77c8e596d6acf1461e3594ced4becedf8921b074fbd8653ed7051c"
+  "packageManager": "pnpm@9.10.0+sha512.73a29afa36a0d092ece5271de5177ecbf8318d454ecd701343131b8ebc0c1a91c487da46ab77c8e596d6acf1461e3594ced4becedf8921b074fbd8653ed7051c",
+  "engines": {
+    "node": "22"
+  }
```

Comment:
can you remove this please? next doesnt support node 22 and is going to throw errors